polyfill/spatial-navigation-polyfill.js

```diff
@@ -973,33 +996,34 @@
   function getInnerDistance(rect1, rect2, dir) {
     const baseEdgeForEachDirection = {left: 'right', right: 'left', up: 'bottom', down: 'top'};
     const baseEdge = baseEdgeForEachDirection[dir];
+
     return Math.abs(rect1[baseEdge] - rect2[baseEdge]);
   }
 
   /**
    * Get the distance between the search origin and a candidate element considering the direction.
    * @see {@link https://drafts.csswg.org/css-nav-1/#calculating-the-distance}
    * @function getDistance
-   * @param rect1 {DOMRect} - The search origin
-   * @param rect2 {DOMRect} - A candidate element
+   * @param element1 {DOMRect || Point} - The search origin
+   * @param element2 {DOMRect} - A candidate element
```

Comment:
These variable name are confused whether it is an element or coordinates set.

Reply:
Yes, it looks like that.
But the parameters would be the elements or points.
So how about changing to "object"?

Reply:
exactly in the function, it could be element't rect or point. 
so I think `element` is not proper.
Hm.. 'points'? or.... coodinates? naming problem is always difficult.

Reply:
I'll fix element1 to searchOrigin, element2 to candidate

Reply:
Good~